lib/src/serializer/repo.dart

```diff
@@ -137,42 +112,112 @@ class SerializerRepo {
     if (object is String || object is num) {
       return object;
     }
-    final Serializer serializer = getByType(type);
 
-    if (serializer == null) {
-      throw new Exception("Cannot find serializer for type $type");
-    }
+    if (object is Map) {
+      final map = {};
+      object.forEach((key, value) {
+        final k = _to(key,
+            type: key.runtimeType, withType: withType, typeKey: typeKey);
+        if (value == null) {
```

Comment:
My understanding is `_to(value)` returns null. Why i put the check there is you have `null.runtimeType` which returns `Null` probably not what we wany. Although, it produces expected results.